workspaces/redhat-resource-optimization/docs/Permissions.md

```diff
@@ -0,0 +1,41 @@
+The redhat-resource-optimization plugin protects its backend endpoints with the builtin permission mechanism and combines it with
+the RBAC plugin.
+
+## redhat-resource-optimization plugin Permissions
+
+| Name       | Resource Type | Policy | Description                                                                                          |
+| ---------- | ------------- | ------ | ---------------------------------------------------------------------------------------------------- |
+| ros.plugin | -             | read   | Allows the user to read from the redhat-resource-optimization plugin and give access to all the data |
+
+## Defining Policy File
+
+To get started with policies, we recommend defining roles and assigning them to groups or users.
+
+As an example, check the following [policy file](../plugins/redhat-resource-optimization-common/rbac-policy.csv).
```

Comment:
our customers deploy RHDH in a container. We need to make sure we provide steps for k8s/ocp as well.